dashboard/public/config.json

```diff
@@ -1,5 +1,7 @@
 {
   "namespace": "default",
   "appVersion": "DEVEL",
+  "authProxyEnabled": false,
+  "loginURI": "/oauth2/start",
   "logoutURI": "/oauth2/sign_out"
```

Comment:
now that I think about this, maybe it's a bit confusing to call this `loginURI` (or `logoutURI`) since kubeapps has alredy URIs for the login (`/login`), maybe it's better to call it `oauthLoginURI`? Feel free to do that in a different PR or ignore it.

Reply:
Yep, will do.